### ISSUE TITLE
Fetch from multiple trust anchors in parallel?

### ISSUE THREAD

**Issue by sraustein** [dragonresearch/rpki.net#105]
Not sure about this, but from watching logs it seems that rcynic is missing opportunities where it could have forked the stack.  In particular, why do we never have anything else we can work on while waiting for NIST?

Might also want to rework so we can move on to the next trust anchor, but that's both a larger change and reversal of a deliberate design decision.  The reason, such as it was, for running trust anchors separately was a belief that we should be all done with one before moving on to the next, so that precedence would be clear, but thinking more about the URL scheme I'm not sure it matters.  If we accept an object, we accept it; if we reject an object with every trust anchor we try it against, we reject it.  Does it matter which one caused us to accept it?   The key point is that it's going to be the same object no matter how many paths we take trying to make it validate, so worrying about trying to postpone investigating some of those paths doesn't seem to buy us much.

But first I want to understand why simple cases like NIST aren't forking when they could.

_Trac ticket #92 component rcynic priority minor, owner sra, created by sra on 2011-09-29T15:00:07Z, last modified 2013-11-22T19:50:03Z_
